project_files/bank_stripped.js

```diff
@@ -0,0 +1,37 @@
+class Printer {     // Responsibility - Print account history
+    printStatement(account) {
+        console.log("Account Summary:\nDate || Credit || Debit || Balance");
```

Comment:
Prefer to return strings and use `console.log()` closer to app entry point - helps decouple app logic from presentation logic

```js
console.log(printer.printerStatement([]))
```